### PR TITLE
Tidy gemini code

### PR DESCRIPTION
**Pull request by chernyee-rs**
Tidy and clean up gemini code

### CODE REVIEW

File: src/pld/gemini.c [hunk 1]
```diff
@@ -14,44 +14,31 @@
...
-#define GEMINI_SPARE_REG      	0xf10000f0
-#define GEMINI_SRAM_ADDRESS   	0x80000000
-#define GEMINI_LOAD_ADDRESS   	0x00000000
+#define GEMINI_SPARE_REG		0x800000f0
+#define GEMINI_SRAM_ADDRESS		0x80001000
+#define GEMINI_LOAD_ADDRESS		0x80001000
+#else
+#define GEMINI_SPARE_REG		0xf10000f0
+#define GEMINI_SRAM_ADDRESS		0x80000000
+#define GEMINI_LOAD_ADDRESS		0x00000000
```
Comment (guanyung-rs) [inline src/pld/gemini.c:26]: they are not aligned

File: src/pld/gemini.h [hunk 2]
```diff
@@ -22,13 +22,12 @@ struct gemini_pld_device {
...
-#define STATUS_IN_PROGRESS		1
-#define STATUS_SUCCESS			2
-#define STATUS_FAIL				3
+#define TASK_COMMAND_IDLE		0
+#define TASK_COMMAND_A			1	/* BootROM to load and jump to FSBL, FSBL to initalize DDR */
+#define TASK_COMMAND_B			2	/* FSBL to program bitstream to QSPI Flash */
+#define TASK_COMMAND_C			3	/* FSBL to program bitstream to FPGA Fabric */
+#define TASK_STATUS_IDLE		0	/* This is mean the command is in progress as well */
+#define TASK_STATUS_SUCCESS		1
+#define TASK_STATUS_FAIL		2
```
Comment (guanyung-rs) [inline src/pld/gemini.h:31]: alignment

**Review by guanyung-rs — COMMENTED** [review]
Just minor formatting issue. Looks good

**Review by esther-rs — APPROVED** [review]

File: src/pld/gemini.c (hunk 1 above)
Comment (chernyee-rs) [reply src/pld/gemini.c:26]: they are aligned. 
image

Comment (chernyee-rs) [reply src/pld/gemini.c:26]: I follow the OpenOCD C coding style as documented here: https://openocd.org/doc-release/doxygen/stylec.html

image

**Review by guanyung-rs — APPROVED** [review]

**Pull request merged**